script/service/google-calendar-webworker.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @author Bogdan Kovalev
+ */
+
+function convertToTask(item) {
+    return {
+        _description: item.summary,
```

Comment:
why names start with underscore. this naming usually used for private js methods and fields